config/fxdata/lua/triggers.lua

```diff
@@ -112,8 +114,6 @@ function TriggerRegisterThingEvent(trigger, thing, thingEvent)
     return event
 end
 
--- Trigger variables
-
 --- Gets the unit associated with the current triggering event
 --- @return Thing|nil
 function GetTriggeringThing()
```

Comment:
This is bad as a concept.
Just pass things from `On...`

Reply:
it's based on the wc3 script, this way allows multiple events to trigger the same action, so can't know the exact params without having a fixed action that triggers it